### ISSUE TITLE
Line 8 (Shelikof Strait) Nutrients, Chl a, Microzooplankton, and Zooplankton data

### ISSUE THREAD

**Issue by clward** [NCEAS/goa-data#28]
(Continued from [Issue 12](https://github.com/NCEAS/goa-data/issues/12))
1985-2014
NOAA EcoFOCI program at Shelikof Strait; run in parallel to Ichthyoplankton sampling

data contacts:
Janet Duffy-Anderson and Colleen Harpold (both NOAA)

Oct 30:  Email conversation with Janet and Colleen.  Colleen will query the databases and send the data to Colette.

**Comment by clward** [comment]
Email exchange with Janet Dec 14 2015:
Colleen is busy until mid-January and will send the data then.

**Comment by clward** [comment]
Update from Colleen Harpold Feb 9 2016:
Extracting the data has been more time consuming than anticipated but she anticipates sending the data by the end of this week.

**Comment by clward** [comment]
Received some of the data from C. Harpold:
1) Chlorophyll data (2004-2012)
2) Nutrient data (2000-2011 (data do not exist for 2002 and 2003)
3) Zooplankton data (1989-2012)

Colleen says: "I still need to get the displacement volume and microzooplankton data. We need to do some adjustments in our database before I can get those data. I will be in touch with those data when I have them."

Data request still outstanding:
Zooplankton abundance (1985-1988, 2013-2014)
Zooplankton displacement volume (1985-2014)
Microzooplankton (all years; not sure how far back these go)

I have asked Janet and Colleen if these datasets can be posted as a data package on the NCEAS AOOS portal.

**Comment by clward** [comment]
Janet has agreed to these data being posted on NCEAS' AOOS site as a data package and has said that she can be the contact person. She would like to work with NCEAS personnel to build the package during the March working group meeting.

**Comment by clward** [comment]
Questions about Zooplankton data:

**emailed Colleen with questions 1-4 Feb 15, 2016** 
1. Is there data going back to 1985?
2. need definitions and/or other info for the following columns:
   FOCI_GRID
   HAUL_PERFORMANCE (what criteria were used to establish questionable vs good, and should questionable data be discarded or just used with caution?)
   VOLUME_FILTERED (what are the units?)
   GMT_DATE_TIME (verifying that these are reported as GMT time zone)
   EST_NUM_PERM2 and EST_NUM_PERM3 (how were these calculated, and what are the units?)
3. Is it appropriate to use the metadata sent a while back to decide which mesh size to use for each zooplankton species and size class? (for example, only the 153um mesh net should be used to derive quantitative estimates of Neocalanus cristatus CII?)
4. only use Euphausiid data collected at night?

**More questions:**
**5.** Is there data for 1995, 1997, 1999, 2002, 2003? (spreadsheet does have any data for these years)

**6.** Spreadsheet has data for 333um mesh (gear_name == 60BON) for 1989-1994, 1996, 1998, 2000, 2001, 2004-2012. Is there 333um data for other years?
Spreadsheet has data for 153um mesh (gear_name == 20BON) for 2002-2012. Is there 153um mesh data for the other years?

**7.** For 1994, there is only data for March? (cruise 3MF94)

**8.** Verify that the dataset includes quantitative zeros. It sounds like it does, except for 2012.

**Comment by clward** [comment]
Answers to questions about Zooplankton dataset (above):

**Email from Colleen Harpold re q1-4, Feb 16, 2016:**
**1.** Yes, we do have data from 1985 to present for all data sets. However, our older chlorophyll and nutrient data sets are not in our current database yet (a project that will happen in the future, but hasn't happened yet). So I provided what is available at this time.

**2.**
**FOCI_GRID** This is our grid name of the station (same lat and long) so it can be repeated year after year.
**HAUL_PERFORMANCE** It is up to you whether you use questionable tows or not. (I recommend not using failed tows) That is an individual decision, some researchers here exclude all questionable tows and others choose whether to include them on a tow by tow basis.. The "Haul Comments" field can be used to determine what was questionable about the tow (and whether or not you want to include it). Unfortunately the database did not allow me to extract the haul comments during my zooplankton search. I will do another search for the haul data and provide the haul comments and you can look them up that way. I will provide this when I can.
**VOLUME_FILTERED** units are meters cubed (M^3) This is the amount of water that the net filtered.
**GMT_DATE_TIME** reported in GMT for time and date.
**EST_NUM_PERM2** = estimated number of organisms per surface area (per meter squared) **EST_NUM_PERM3** = estimated number of organisms per volume of water (per meter cubed) Typically people in our group express zooplankton data per volume of water (EST_NUM_PERM3).

**3.** correct

**4.** I'm not sure what the standard is for reporting euphausiid data. I know they are notoriously hard to sample and there can be day / night effects.

**I found answers to q5 & 6 by looking at NOAA's online metadata:**
**5.** from metadata posted online here: http://access.afsc.noaa.gov/icc/selectionframe.php 
1995: it looks like we should have cruise 8MF95 (60BON, stns 96-101, May 20-29)
1997: it looks like Line 8 was not sampled
1999: it looks like we should have cruises 1WE99 (May 7-9, stns 1-6) and 5MF99 (May 21 - June 2, did not sample Line 8 but stns 95 & 96 look close)
2002, 2003: no EcoFOCI sampling in these years

**6.** from metadata posted online here: http://access.afsc.noaa.gov/icc/selectionframe.php 
153um mesh net was not fished before 2002. Only 333um mesh data are available before 2002.

**Comment by clward** [comment]
Zooplankton cleaning script is here:  https://github.com/NCEAS/dmx-common/blob/master/Line8Zooplankton/Line8ZooplanktonCleaningScript.R

**Comment by clward** [comment]
[EcoFOCI Line 8_descriptor.docx](https://github.com/NCEAS/goa-data/files/133901/EcoFOCI.Line.8_descriptor.docx)
Attached file describes lower food web time series collected at Line 8 by EcoFOCI program. File author is Colleen Harpold (NOAA)

**Comment by clward** [comment]
sampling protocols are in this document:
http://www.afsc.noaa.gov/Publications/ProcRpt/PR2010-02.pdf

for CTD data, refer to "CTD Cast"

**Comment by clward** [comment]
use abstract from here: 
http://www.pmel.noaa.gov/foci/FOCI_about.html

**Comment by couture322** [comment]
Data contact and PI: Janet Duffy-Anderson

Usage rights: Janet wants the usage rights to request that users contact her prior to use but list that it is public data so they can use it as they like. See the [herring ASWL dataset](https://goa.nceas.ucsb.edu/#view/df35b.273.7) and [NOAA groundfish dataset](https://goa.nceas.ucsb.edu/#view/df35b.258.10) for text, combine and/or use one of these but site NOAA AFSC.

Abstract: use first paragraph from link above to start. Second paragraph should highlight that this data set is the updated version as of 2016 and the data that were used by the PFX working group. Give a short description of the working group and direct to ecofoci and ASFC for newer data and more information.

Final posting should have a DOI so that this version can be sited.

**Comment by couture322** [comment]
See [Colette's scripts](https://github.com/NCEAS/dmx-common/tree/master/Line8Zooplankton) for links to the Zooplankton data file

I'll also email you the nutrients, phytoplankton (and duplicate of above)

**Comment by couture322** [comment]
Colette says that the datasets I sent you are not the _complete_ sets but we can use these initially to create the metadata and post what we have then update the files once we get the rest of the data (since the columns should all be consistent).  

As for your emailed questions:
- In Colette’s code she made a processed dataset that is the mean spring abundance for microzooplankton (I think this one she made it for the working group) and then also one unprocessed raw data for all time periods. Which one should be the one uploaded, or both?
  - **Just use the original data, anyone else can easily take averages from that if they need. I guess that takes care of the rest of your questions**
- In the mean spring abundance for micro zooplankton, there are two microzookplanton species (Ctenophora, NatantiaAdJuv) that have zero abundance, am not sure if its an error? Should I check the code?
  - **Just use the original data**
- Silly question, but is microzooplankton and zooplankton the same….?
  - **Microzooplankton are zooplankton that are a certain size so if you see both, you can assume the micro are the small ones and the regular zooplankton are bigger ones. The metadata should define what size the micro are.**

**Comment by sandysum** [comment]
Have created metadata and added sampling protocol and methods from EcoFOCI Field Manual provided: http://www.afsc.noaa.gov/Publications/ProcRpt/PR2010-02.pdf

Uploaded zooplankton data table
Midway through defining columns for nutrient and chlorophyll tables,need to clarify some column variables
**For Nutrient data table:**
- dilution factor
- questuestuality - I assume it means "performance" as factor values are similar to that in zoo.p data column but the rows seem incomplete
- columns: NH4, NO2, NO3, PO4, SIO4, assume these are nutrients but need units for them

**For Chlorophyll data table:**
- F_B
- F_A
- Acetone volume
- dilution factor
- sample order
- tau
- calibration id

**Comment by sandysum** [comment]
Hi @clward!

I'm assisting Jessica with the cleaning, metadata creation and uploading of the Chlorophyll, Nutrients and Zooplankton data tables to the GOA Data Portal. 

I have been following the format you specified in your zooplankton cleaning R script and adapted accordingly for the other two tables.

Have some questions regarding some of the variables in the data table below that perhaps you could help with. 

**For Nutrient data table:**
- dilution factor (gave generic definition of ratio of volume to volume diluted, but of what substances?)
- columns: NH4, NO2, NO3, PO4, SIO4. Need units for these nutrients. Are they concentrations, or in moles, or molecular density etc.?
- observations for performance variable is either empty or "quest", do we assume "good" for empty rows?

**For Chlorophyll data table:**
- F_B
- F_A
- Acetone volume, collected or added?
- PHAECONC - not sure what is this? Concentration of Phaeocystic algae, which is cited as a problem in contamination of the sample?
- dilution factor (same as above)
- volume filtered, I defined it as
  "Total volume filtered for batch of microcentrifuge tube from each CTDB cast Niskin bottle", but am not entirely sure if it is total volume filtered or per tube or per batch...
- sample order (do we need this and the following in the AOOS data?)
- slope
- tau
- calibration id

**Comment by sandysum** [comment]
First draft saved on network on GOA Data Portal: https://goa.nceas.ucsb.edu/#view/df35h.13.2
